web/src/components/storage/VolumeForm.jsx

```diff
@@ -70,12 +71,39 @@ const SizeUnitFormSelect = ({ units, ...formSelectProps }) => {
  * @param {Array<import(~/clients/storage).Volume>} props.volumes - a collection of storage volumes
  * @param {object} props.formSelectProps - @see {@link https://www.patternfly.org/components/forms/form-select#props}
  * @returns {ReactComponent}
- */
-const MountPointFormSelect = ({ volumes, ...formSelectProps }) => {
+*/
+
+const MountPointFormSelect = ({ volumes, ...selectProps }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [chosen, setchosen] = useState(false);
+
+  const onSelect = () => {
+    setIsOpen(false);
+    setchosen(!chosen);
+  };
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const toggle = toggleRef => {
+    return (
+      <Button ref={toggleRef} onClick={onToggleClick}>
+        {chosen ? chosen.mountPath : 'choose atleast one volume'}
+      </Button>
```

Comment:
I prefer using the `MenuToggle` as you can see in the [`Select` documentation examples](https://www.patternfly.org/components/menus/select). 

Regarding default value, I'd use something more simple like _Select a value_ or _Select a mount point_, no strong opinion. In any case, Agama has support for both, [i18n and l10n](https://en.wikipedia.org/wiki/Internationalization_and_localization), so please use the `_` function. I.e., `_("Select a value")` instead of `"Select a value"`. Read more at https://github.com/openSUSE/agama/blob/master/doc/i18n.md#marking-texts-for-translation